src/app/app-winui3/InputSettingsPage.xaml

```diff
@@ -71,6 +71,44 @@
         Text="Input Settings"
         Margin="0,0,0,8"
       />
+      <TextBlock 
+        Text="Pen" 
+        Style="{StaticResource SubtitleTextBlockStyle}" 
+        Margin="0,0,0,16" />
+      <StackPanel 
+        Background="{ThemeResource CardBackgroundFillColorDefaultBrush}" 
+        BorderBrush="{ThemeResource CardStrokeColorDefaultBrush}" 
+        CornerRadius="4"
+        BorderThickness="1"
+        Padding="8"
+        Margin="0,0,0,8">
+        <Slider 
+          Header="Write Radius" 
+          Value="{x:Bind WriteRadius, Mode=TwoWay}" 
+          TickPlacement="Outside" 
+          TickFrequency="10" 
+          StepFrequency="1" 
+          Maximum="100" 
+          Minimum="1.0" />
+        <ToggleSwitch 
+          Header="Fixed Write Radius" 

```

Comment:
Perhaps invert these (just in the UI), and relabel as something like "Pressure-sensitive pen stroke size" + "Pressure-sensitive eraser size" ?